test_definitions.py

```diff
@@ -114,6 +118,25 @@ def test_filename_matches_type_version_fields(definition_file):
     assert version == definition_file.definition["_version"]
 
 
+@pytest.mark.parametrize(
+    "definition_file",
+    EVENT_DEFINITIONS + OTHER_DEFINITIONS,
+    ids=EVENT_DEFINITION_IDS + OTHER_DEFINITION_IDS,
```

Comment:
Since the copied meta schema is 2020-12 I believe it makes sense to only validate the recent schemas and not all old ones. With that you could remove the misspelled additonalProperties from the schema I assume. Otherwise there is a risk that it will be re-added again in a future definition file version.

```suggestion
    EVENT_DEFINITIONS_W_LINKS_REQUIRED + OTHER_DEFINITIONS,
    ids=EVENT_DEFINITION_W_LINKS_REQUIRED_IDS + OTHER_DEFINITION_IDS,
```

Reply:
Fixed in feb501c